tensornetwork/backends/numpy/numpy_backend.py

```diff
@@ -135,14 +136,136 @@ def randn(self,
       self.np.random.seed(seed)
 
     if not dtype:
-      dtype = (self.dtype if self.dtype is not None
-               else self.np.dtype(self.np.float64))
+      dtype = (
+          self.dtype
+          if self.dtype is not None else self.np.dtype(self.np.float64))
 
-    if ((dtype is self.np.dtype(self.np.complex128))
-        or (dtype is self.np.dtype(self.np.complex64))):
+    if ((dtype is self.np.dtype(self.np.complex128)) or
+        (dtype is self.np.dtype(self.np.complex64))):
       return self.np.random.randn(*shape).astype(
           dtype) + 1j * self.np.random.randn(*shape).astype(dtype)
     return self.np.random.randn(*shape).astype(dtype)
 
   def conj(self, tensor: Tensor) -> Tensor:
     return self.np.conj(tensor)
+
+  def eigsh_lanczos(
```

Comment:
Be sure to add this to `BaseBackend` too

Reply:
added eigsh_lanczos to all other backends.